### PR TITLE
Fix a bunch of warnings

### PR DESCRIPTION
**Pull request by bhariharan**

### CODE REVIEW

File: shared/resources/Images.xcassets/AppIcon.appiconset/Contents.json [hunk 1]
```diff
@@ -113,12 +113,6 @@
       "idiom" : "ipad",
       "filename" : "appicon152x152.png",
       "scale" : "2x"
-    },
-    {
-      "size" : "83.5x83.5",
```
Comment (khawkins) [inline shared/resources/Images.xcassets/AppIcon.appiconset/Contents.json]: How come we're taking this out?  It's required for iPad apps running iOS 9.

Comment (bhariharan) [inline shared/resources/Images.xcassets/AppIcon.appiconset/Contents.json]: We don't have those images, so there's an `unassigned children` warning generated.

File: shared/resources/Images.xcassets/AppIcon.appiconset/Contents.json [hunk 2]
```diff
@@ -113,12 +113,6 @@
...
-      "idiom" : "ipad",
-      "filename" : "appicon167x167.png",
```
Comment (khawkins) [inline shared/resources/Images.xcassets/AppIcon.appiconset/Contents.json]: The deleted file below?  Is it just in the wrong place, wrong reference?  The file exists....

**Comment by bhariharan** [comment]
appicon

@khawkins Looks like the required assets are there. There was an extra `unassigned` child that I removed.

**Comment by bhariharan** [comment]
unassigned

@khawkins Here's the `unassigned` screenshot.

**Comment by khawkins** [comment]
What version of Xcode are you running?  That AppIcon set doesn't include the iPad Pro, which is what the image is targeted for.

**Comment by bhariharan** [comment]
`Xcode 7.1`.

**Comment by khawkins** [comment]
Yeah, that'll do it. ;-)  In general, I'd recommend updating to 7.3.1, since you'll undoubtedly find a bunch of additional warnings there too. ;-)  Xcode keeps upping its warnings to handle the imminent Swift compatibility requirements.

**Comment by bhariharan** [comment]
Cool, will do. Meanwhile, should I just close this PR?

**Comment by khawkins** [comment]
The other stuff looks fine.  Maybe just revert the image assets updates, then merge it?

**Comment by bhariharan** [comment]
Cool, will do.

**Comment by bhariharan** [comment]
@khawkins Done.

**Comment by khawkins** [comment]
👍

**Pull request merged**